### ISSUE TITLE
Embedcode Dialog needs more text for screen readers

### ISSUE THREAD

**Issue by rlucke** [opencast/opencast-admin-interface#1272]
![Image](https://github.com/user-attachments/assets/08d29a9a-6861-492a-9b83-c87026ab89ea)

Only the dimensions of the videos at the counters are read out here. It is not clear what pressing the switch does. 
The field in which the embed code is entered also has no labelling, so it is unclear what the field is used for.

Overall, the dialogue could do with more labels or explanations. Even if it is an admin interface, the UI should be as self-explanatory as possible.
